webapp/src/widgets/icons/logo.tsx

```diff
@@ -10,7 +10,7 @@ export default function LogoIcon(): JSX.Element {
     return (
         <CompassIcon
             icon='product-boards'
-            className='LogoIcon'
+            className='boards-rhs-icon'
```

Comment:
Accordingly to our policy is always needed a class name with the component name, in this case `LogoIcon`, why do you need an specific class for this?

Reply:
@jespino Because it conflicts with the CSS in the webapp otherwise. It should be super specific here to not pollute.

Reply:
I would prefer changing the component name a keeping the "class by component" rule. I can do it after merge, so, don't worry.